packages/st2mistral/debian/mistral-api.init

```diff
@@ -69,6 +70,18 @@ do_start()
   #   1 if daemon was already running
   #   2 if daemon could not be started
   lsb_running && return 1
+
+  # /run is a tmpfs mount, so we need to check the directory exists and create.
+  # Otherwise the service will not start following a reboot.
+  if [ ! -d "$RUN_DIR" ] ; then
+      if ! mkdir -p "$RUN_DIR" ; then
+          log_failure_msg "can't create $RUN_DIR"
+          exit 1
+      fi
+      if ! chown mistral:mistral "$RUN_DIR" ; then
```

Comment:
Group is `root`, from https://github.com/StackStorm/st2-packages/blob/da3b20de4f454f77c87632b15fd46b9a46e03f7f/packages/st2mistral/debian/postinst#L27